src/sagemaker_training/mpi.py

```diff
@@ -341,6 +341,18 @@ def _create_command(self):
             command.extend(["-x", "FI_PROVIDER=efa"])
             # Use simple protocol to handle the out-of-order data delivery from EFA
             command.extend(["-x", "NCCL_PROTO=simple"])
+            # Further set NCCL_ALGO for modelparallel job to avoid hang, starting from torch2.0.0
+            env = environment.Environment()
+            if env.is_modelparallel_enabled:
+                try:
+                    import torch
+
+                    if torch.__version__.split(".")[0] != "1":
```

Comment:
- Do you mean this is only work for pt 2.0? can this line be more interpretable like `greater than 1.x`
- Can we add some comments or doc to indicate why we need to do so?

Reply:
Updated.